src/script/ptl-tool.py

```diff
@@ -70,6 +70,8 @@
 import sys
 
 from os.path import expanduser
+from email.utils import parseaddr
+from email_validator import validate_email, EmailNotValidError
```

Comment:
i'd suggest avoid introducing this dependency.

Reply:
@tchaikov I had checked out a few email validators and this seems to be the better one. Do you suggest any other ?

Reply:
@tchaikov Another point is: This doesn't affect the users of Ceph, as the script is going to be used only by `wip-*-testing` people.

Reply:
I don't really want to keep adding obscure dependencies if we can avoid it. I think just relying on the PTL to self-validate `Reviewed-by`s is fine.